### DISCUSSION TITLE
Why is plausible using erlang 24.3.3 over 24.3.4

### DISCUSSION THREAD

**Discussion by susanthenerd** [plausible/analytics#2483] [Anything Else]
Hello, 
Due to some issue(I can't install it with asdf) with erlang 24.3.3, I have to use with 24.3.4. Is there any reason why it is set to the older version?

Thanks,
Cristi

**Comment by ukutaht** [comment]
Just haven't updated it yet. Not a conscious choice for any specific reason.

**Reply by susanthenerd** [reply]
Can you update it? If not, I will just set it back to 24.3.3 before I do my PR.

**Reply by ukutaht** [reply]
It will take time to test it out in pre-prod before pushing it to prod. I'd recommend just doing your PR with the current version
